### PR TITLE
Remove dependency of canvas_traits on azure

### PR DESCRIPTION
**Pull request by rohithmukku**
---

- [ ] `./mach build -d` does not report any errors
- [ ] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

Signed-off-by: Rohith Mukku <rohithmukku@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14547)

### CODE REVIEW

**Comment by Naruto8** [comment]
This is my first time, if there are any mistakes or errors... help me rectify it! Thanks!

**Comment by KiChjang** [comment]
What issue number is this trying to solve?

File: components/canvas/canvas_paint_thread.rs [hunk 1]
```diff
@@ -864,3 +864,180 @@ impl ToAzFloat for Rect<f64> {
...
+
+
+pub trait ToAzureStyle {
+    type Target;
+    fn to_azure_style(self) -> Self::Target;
+}
+
+impl ToAzureStyle for FillOrStrokeStyle {
+    type Target = Option<Pattern>
+    fn to_azure_pattern(&self, drawtarget: &DrawTarget) -> Self::Target {
```
Comment (KiChjang) [inline components/canvas/canvas_paint_thread.rs]: Have you tried compiling your changes? This does not(look like it should compile.

File: components/canvas/canvas_paint_thread.rs [hunk 2]
```diff
@@ -864,3 +864,180 @@ impl ToAzFloat for Rect<f64> {
...
+        }
+    }
+}
+
+pub trait ToString {
+	fn to_string(self) -> &str;
+}
+
+impl ToString for CompositionStyle {
+	fn to_str(&self) -> &str {
```
Comment (KiChjang) [inline components/canvas/canvas_paint_thread.rs]: This also does not look like it should compile.

File: components/canvas/canvas_paint_thread.rs [hunk 3]
```diff
@@ -864,3 +864,180 @@ impl ToAzFloat for Rect<f64> {
...
+            CompositionStyle::DestAtop => "destination-atop",
+            CompositionStyle::Copy     => "copy",
+            CompositionStyle::Lighter  => "lighter",
+            CompositionStyle::Xor      => "xor",
+        }
+    }
+}
+
+impl ToString for BlendingStyle {
+	fn to_str(&self) -> &str {
```
Comment (KiChjang) [inline components/canvas/canvas_paint_thread.rs]: Same here.

**Review by KiChjang — CHANGES_REQUESTED** [review]

**Comment by KiChjang** [comment]
@Naruto8 Are you still planning on working on this?

**Comment by KiChjang** [comment]
Closing due to inactivity.

**Pull request closed without merging**